frontend/ui/widget/sortwidget.lua

```diff
@@ -475,6 +481,10 @@ function SortWidget:onSwipe(arg, ges_ev)
     end
 end
 
+function SortWidget:onShowWidgetMenu()
+    self:showMenu()
+end
+
 function SortWidget:showMenu()
```

Comment:
I missed what @hius07 noted. It would indeed be better to remove these lines to keep a better overview.

Reply:
That was noted in https://github.com/koreader/koreader/pull/11916#discussion_r1621052538, but disappeared quickly under the Resolve conversation.

Reply:
GH frequently auto-resolves things if you change a few lines of code. It's a bit annoying. Reviewable keeps better track of it I think.